docs/user/azure/encryption.md

```diff
@@ -0,0 +1,115 @@
+# Azure Disk Encryption with Customer Managed Keys
+
+The installer can use an existing disk encryption set. The disk encryption set should be created in a separate resource group than the installer to avoid losing access to your keys. 
+
+### Set basic variables
+```
+export RESOURCEGROUP="<Resource Group>"
+export LOCATION="<Location>"
+export KEYVAULT_NAME="<Keyvault Name>"
+export KEYVAULT_KEY_NAME="<Keyvault Key Name>"
+export DISK_ENCRYPTION_SET_NAME="<Disk Encryption Set Name>"
+export CLUSTER_SP_ID="<Service Principal ID>"
+```
+
+### Register features
```

Comment:
If I understand this correctly,

"Encryption at host is in preview, and must be enabled to be used."

Reply:
Fixed.